tests/test_cursor_description.py

```diff
@@ -167,13 +168,76 @@ def test_types(self):
                 else: # pragma: nocover
                     pass
 
+    def test_multiple_descriptions(self):
+        with self.connect() as connection:
```

Comment:
in `TestCursorCallproc` there's a `stored_procedure` function to create/drop a sproc. Probably should factor that out and use it here. Also that function itself appears to need a `try: except:` around the `yield`

Reply:
Done.